### PR TITLE
Move tx-queue/acquire-lock call to process-transactions

### PR DESCRIPTION
**Pull request by cap10morgan**
The expected args aren't available where it was being called before.

Fixes https://github.com/fluree/core/issues/64

### CODE REVIEW

**Comment by cap10morgan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #20** <a href="https://app.graphite.dev/github/pr/fluree/server/20?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #21** <a href="https://app.graphite.dev/github/pr/fluree/server/21?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #22** <a href="https://app.graphite.dev/github/pr/fluree/server/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/fluree/server/22?utm_source=stack-comment).

**Review by zonotope — APPROVED** [review]
🍉

File: src/fluree/server/consensus/handlers/tx_queue.clj [hunk 1]
```diff
@@ -162,19 +162,20 @@
          processed (list)] ;; list of transactions that have been evaluated
     ;; TODO - push-consensus will only work if a leader. As leader can only do txns for the moment this is OK
     ;; TODO - but once the work gets spread out to other servers it will have to push the commit to the leader
-    (let [processed*     (conj processed tx-id)]
+    (when (acquire-lock ledger-id tx-id)
```
[the remark this answers is not in the document]
Comment (cap10morgan) [reply src/fluree/server/consensus/handlers/tx_queue.clj:165]: I'm not sure, to be honest. But where it was being called before if the lock acquisition failed it wouldn't even call this fn, which I think is basically the same behavior it has now.

**Comment by cap10morgan** [comment]
### Merge activity

* **Nov 21, 1:21 PM**: @cap10morgan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/fluree/server/22).

**Pull request merged**